app/app.rb

```diff
@@ -282,7 +282,7 @@ def render_directory(directory)
 		def process_form_data(data, git)
 			case data
 			when Array
-				data.map { |v| process_form_data(v, git) }
+				data.reject { |v| v.is_a?(String) && v.blank? }.map { |v| process_form_data(v, git) }
```

Comment:
What happens if the array is empty?  Hopefully YAML emits some kind of helpful empty array.